velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -52,7 +54,8 @@ struct HiveConnectorSplit : public connector::ConnectorSplit {
         partitionKeys(_partitionKeys),
         tableBucketNumber(_tableBucketNumber),
         customSplitInfo(_customSplitInfo),
-        extraFileInfo(_extraFileInfo) {}
+        extraFileInfo(_extraFileInfo),
+        options(readerOptions) {}
```

Comment:
options(std::move(readerOptions)) with the above change.

Reply:
Also, where are options used in the HiveConnectorSplit?

Reply:
Nowhere, removed.